src/status_im2/common/bottom_sheet/view.cljs

```diff
@@ -74,9 +74,6 @@
             bottom                  (if sheet-bottom-margin
```

Comment:
I think @jo-mut was working on fixing the extra space at the bottom?

Reply:
this pr. addresses the issue https://github.com/status-im/status-mobile/pull/17547

Reply:
Thanks for the update @jo-mut.

I checked your PR. It resolves the bottom spacing, but the background color on the shell (dark blur) theme and gradient cover is fixed in this PR. 

I guess, I should wait for your PR to get merged. To include your fixes in this PR.

Reply:
thats fine with me thank @smohamedjavid

Reply:
@smohamedjavid https://github.com/status-im/status-mobile/blob/43c73d849c64d3c2b083a26c8e395adad71dd8bd/src/status_im2/common/bottom_sheet/view.cljs#L73

removing this code from the following line will solve the issue on the extra margin`(:bottom insets)`

Reply:
and also remove `(:bottom insets)`  in this line too and that should fix the issue in this pr
https://github.com/status-im/status-mobile/blob/43c73d849c64d3c2b083a26c8e395adad71dd8bd/src/status_im2/common/bottom_sheet/view.cljs#L74

> (:bottom insets)

Reply:
Thanks @jo-mut - I will add the fix in this PR 👍